spec/features/state_file/complete_intake_spec.rb

```diff
@@ -328,113 +329,39 @@
       expect(submission.submission_bundle).to be_present
       expect(submission.current_state).to eq("queued")
     end
-
```

Comment:
removed this and put in `nc/sales_use_tax_spec.rb`

Reply:
i was looking at this recently and wondering "huh, this should not be a feature spec" Ty for moving!